openedx/core/djangoapps/embargo/api.py

```diff
@@ -170,10 +170,15 @@ def _country_code_from_ip(ip_addr):
         str: A 2-letter country code.
 
     """
+    reader = geoip2.database.Reader(settings.GEOIP_PATH)
     if ip_addr.find(':') >= 0:
```

Comment:
Do we need if else here? I don't think it's doing anything different, because now we have same database for both IPV4 and IPV6.